models/system/bert-base-cased/spec.yaml

```diff
@@ -6,6 +6,9 @@ properties:
   datasets: bookcorpus, wikipedia
   finetuning-tasks: text-classification, token-classification, question-answering
   languages: en
+  recommended-sku: Standard_ND40RS_v2
+  minimum-sku: Standard_NC6
+  finetune-min-sku-spec: 6|1|56|380
```

Comment:
For finetune-min-sku-spec, numbers look confusing. Why not add what these numbers represent? (CPU/GPU/RAM/Disk)

Reply:
We can add the additional labels for the values. UI just needs to strip them off to do comparisons. This isn't really a big problem in the UI to do, we were just thinking for convenience

Reply:
From the UI we're not even going to allow the user to choose a sku that doesn't meet the requirements

Reply:
Can we go without units for now? The UI is already implemented without them. We could add them in the future, but as we're only doing 1 UI release a week, it will take another week to add the units and get deployed. It'll be about a 2 week turnaround to get to prod